src/Catty/Functions/Protocols/CBFunction.swift

```diff
@@ -58,26 +58,46 @@ extension CBFunction {
     }
     
     func nameWithParameters() -> String {
-        var name = type(of: self).name
+        var functionHeader = type(of: self).name
+        let functionName = type(of: self).name
         let params = self.parameters()
         var count = 0
         
         if params.count == 0 {
-            return name        // no parameter function
+            return functionHeader       // no parameter function
         }
         
-        name += type(of: self).bracketOpen
+        functionHeader += type(of: self).bracketOpen
         for param in params {
-            name += param.defaultValueString()
+            
+            // add apostrophe at the beginning of a string
+            if ((functionName == "join" && (count == 0 || count == 1)) ||
```

Comment:
Please do not hardcode any function names here. Maybe you can add a new method to the _FunctionParameter_ directly and let the parameter decide whether to add apostrophes or not (can be added to every string parameter). Maybe something like _defaultValueDisplayString()_ ?